### DISCUSSION TITLE
Extra comma appears in a bib entry with just two authors

### DISCUSSION THREAD

**Discussion by pareek21** [alshedivat/al-folio#2811] [Q&A] [answered]
Here is an example from the default Einstein page. Any ideas on how to fix this?

<img width="435" alt="image" src="https://github.com/user-attachments/assets/21c5a04d-b744-4cb8-93e7-a639f5f842bf">

**Comment by george-gca** [comment, marked as answer]
Probably this line is the responsible

https://github.com/alshedivat/al-folio/blob/47ed26fb4ae3bdad22adcb3e23f31c40b9a90961/_layouts/bib.liquid#L81

Looking at it it adds the comma if it is not the first author, but nowhere it looked if there are only 2. Would you mind sending a PR for this?
